salestation.gemspec

```diff
@@ -4,7 +4,7 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 
 Gem::Specification.new do |spec|
   spec.name          = "salestation"
-  spec.version       = "3.2.2"
+  spec.version       = "3.3.0"
   spec.authors       = ["SaleMove TechMovers"]
```

Comment:
Rename to Glia?

Reply:
https://github.com/salemove/salestation/pull/36/commits/4a70df9237475bcbcaafbe5729d1f096f388e922